sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -408,3 +408,13 @@ case class DescribeFunction(
     }
   }
 }
+
+case class SetDatabaseCommand(databaseName: String) extends RunnableCommand {
```

Comment:
We are introducing the concept of a database change in Catalyst. Shouldn't we also have ```SetDatabaseCommand``` in Catalyst?